portfolio/src/main/java/com/google/sps/servlets/Form2.java

```diff
@@ -0,0 +1,5 @@
+package com.google.sps.servlets;
+
+public class Form2 {
+    String name;
```

Comment:
I think there are more variables in addition to "name"?

Reply:
Actually its only name because the feature ask the user for their name to display back in the page along side a hello text.